spec/xclarity_client_update_policy_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'spec_helper'
```

Comment:
Please, could you also change this file name to `xclarity_client_compliance_policy_spec.rb`

Reply:
updated